src/blender_mcp/server.py

```
@@ -11,6 +11,7 @@
 from pathlib import Path
 import base64
 from urllib.parse import urlparse
+import math
```

Ruff (0.8.2)

`math` imported but unused

Remove unused import: `math`

(F401)